src/main.cpp

```diff
@@ -430,6 +519,31 @@ int main(const int argc, const char *argv[])
     inset_state.destroy_fftw_plans_for_rho();
     inset_state.ref_to_rho_init()->free();
     inset_state.ref_to_rho_ft()->free();
+
+    // Calculate distances
+    std::map<std::string, double> hausdorff_distances =
+      inset_state.get_geo_div_differences(hausdorff_distance, true);
+    double total_distance = 0;
+    std::cout << "\nHausdorff distances:\n";
+    for (const auto &[key, val] : hausdorff_distances) {
+      std::cout << key << ": " << val << std::endl;
+      total_distance += val;
+    }
+    std::cout << "\nAverage Hausdorff distance: "
+              << total_distance / double(hausdorff_distances.size())
+              << std::endl;
+
+    std::map<std::string, double> symmetric_distances =
+      inset_state.get_geo_div_differences(symmetric_distance, true);
+    total_distance = 0;
+    std::cout << "\nSymmetric distances:\n";
+    for (const auto &[key, val] : symmetric_distances) {
+      std::cout << key << ": " << val << std::endl;
+      total_distance += val;
+    }
+    std::cout << "\nAverage symmetric distance: "
+              << total_distance / double(symmetric_distances.size())
+              << std::endl;
```

Comment:
Should std::cerr be reserved for errors?

Reply:
Ideally yes. But, since we need to consider the web compatibility as well, it is best we only preserve `std::cout` for writing cartogram GeoJSON, and everything else to `std::cerr`.